types/stripe-js/elements-group.d.ts

```diff
@@ -71,6 +73,25 @@ export interface StripeElements {
    */
   getElement(elementType: 'address'): StripeAddressElement | null;
 
+  /////////////////////////////
+  /// paymentMethodMessaging
+  /////////////////////////////
+
+  /**
+   * Creates an `paymentMethodMessaging`.
```

Comment:
Should this be
```
Creates a `paymentMethodMessaging` element.
```
?

Reply:
Ah good catch, updated to
```
Creates a `paymentMethodMessagingElement`
```

To match the format of other elements